tests/fixtures/update/create.json

```diff
@@ -1,5 +1,40 @@
 {
   "type": "create",
-  "suffix_data": "eyJkZWx0YV9oYXNoIjoiRWlDTW5jb0xrODlIeWpUYzhRbUU0VVVoNFpzMlhiOGlGTWxBak51WFlDbjZCdyIsInJlY292ZXJ5X2NvbW1pdG1lbnQiOiJFaUF5VklubUZfbnpwYWVmeVdqNkJvTkdkWV85RlJOZWYtV0kzY3JySzZ2SXlBIn0",
-  "delta": "eyJ1cGRhdGVfY29tbWl0bWVudCI6IkVpQ2prREdERE9udVpxNEtpdzJpY3VkSU45NWJvRE1yR2RZeDRwejhiQnQyYnciLCJwYXRjaGVzIjpbeyJhY3Rpb24iOiJyZXBsYWNlIiwiZG9jdW1lbnQiOnsicHVibGljX2tleXMiOlt7ImlkIjoic2lnbmluZ0tleSIsInR5cGUiOiJFY2RzYVNlY3AyNTZrMVZlcmlmaWNhdGlvbktleTIwMTkiLCJqd2siOnsia3R5IjoiRUMiLCJjcnYiOiJzZWNwMjU2azEiLCJ4IjoiSk5PMVhsRGVRNDhxUHhrNkFNbjZFNjMzcWJYdTVXS19uX05rZjBvNWtXYyIsInkiOiJRd1BwQkp1NXRQWkhpMml5VzlZbXRtWWl0YzdiS29ObmNCazhSYWc2RVNRIn0sInB1cnBvc2UiOlsiYXV0aCIsImdlbmVyYWwiXX1dLCJzZXJ2aWNlX2VuZHBvaW50cyI6W3siaWQiOiJzZXJ2aWNlRW5kcG9pbnRJZDEyMyIsInR5cGUiOiJzb21lVHlwZSIsImVuZHBvaW50IjoiaHR0cHM6Ly93d3cudXJsLmNvbSJ9XX19XX0"
+  "suffix_data": {
+    "delta_hash": "EiBtq6E2h8k82eh3ZFA8quDB6mD8wfwarQ44dqQBbJnhhw",
+    "recovery_commitment": "EiAPN6FqOS487qru2NgQmnu6zFOTeOaPsH7HbGQmLmUZpQ"
+  },
+  "delta": {
+    "update_commitment": "EiDuOprunhX2vM9ZOSY4M-C7QUQDGSe_g3AKJGoOPMpevA",
+    "patches": [
+      {
+        "action": "replace",
+        "document": {
+          "public_keys": [
+            {
+              "id": "signingKey",
+              "type": "EcdsaSecp256k1VerificationKey2019",
```

Comment:
Prefer JsonWebKey2020

Reply:
Not going to address this in this pr since this pr is purely for SIP 2 changes.